src/languages/csharp.js

```diff
@@ -18,10 +18,13 @@ function(hljs) {
       'uint ulong unchecked unsafe ushort using virtual void volatile while ' +
       // Contextual keywords.
       'add alias ascending async await by descending dynamic equals from get global group into join ' +
-      'let nameof on orderby partial remove select set value var when where yield',
+      'let nameof on orderby partial remove select set value var when where yield ' +
+      // generic modifiers
+      'in out',
```

Comment:
Why do you need them here, outside of `GENERIC_MODIFIER`?

Reply:
I usually try to add keywords in both places (I think we have precedent for this in other grammars).  I don't 100% know for sure they are purely contextual keywords.

Reply:
Yeah, looking into https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/lexical-structure I see that these are rightful keywords. But we already have them above, in "normal keywords" section (:

Reply:
Removing.